scripts/run_backend_tests.py

```diff
@@ -367,6 +397,8 @@ def check_test_results(tasks, task_to_taskspec, generate_coverage_report):
             )
 
             try:
+                if not tests_failed_regex_match:
```

Comment:
Raising an attribute error since there is an exception handler below to handle if there are no matches

Reply:
Please add some description to the errors, I know it is going to be lost when we handle it but it still make sense for readability.

Reply:
No I meant do `Exception('<description>')`

Reply:
done